### DISCUSSION TITLE
"strict" type checking isn't accessible through the status bar at the bottom of the VS code window

### DISCUSSION THREAD

**Discussion by Coniferish** [microsoft/pylance-release#4888] [Enhancement]
<h3> Environment data </h3>

-   Language Server version: 2023.9.20
-   OS and version: darwin arm64
-   python.analysis.indexing: true
-   python.analysis.typeCheckingMode: strict

<h3> Repro Steps </h3>

I'm unable to change the `type checking` level to `strict` from the status bar at the bottom of the VS code window.

1. Set "python.analysis.typeCheckingMode": "strict" (for both user and workspace settings or apply to all profiles)
2. Hover over the Type Checking option in the status bar. (optionally, pin it and hover over the pinned option)
3. Toggle type checking on/off

<h3> Expected behavior </h3>

I'd expect type checking to toggle between "strict" and "off" after setting it

<h3> Actual behavior </h3>

Selecting the type checking option on the status bar toggles between "off" and "basic" (without the ability to select "strict")

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
